src/layouts/Layout.astro

```diff
@@ -14,6 +14,16 @@ const { title } = Astro.props;
 		<meta name="viewport" content="width=device-width" />
 		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
 		<meta name="generator" content={Astro.generator} />
+
+		<!--
+			font-family
+		 -->
+
+		 <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,100;1,400;1,700;1,900&family=Plus+Jakarta+Sans:ital,wght@0,200;0,300;0,400;1,200;1,300&family=Poppins:ital,wght@0,200;1,200&family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+     
```

Comment:
Properly indent the code. Use Astro's VS Code extension to auto-format the code for constitency.